app/services/valkyrie1_migrator.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+# Service Class for migrating from Valkyrie 0.x releases to 1.0
+# Please note that this approach should not be undertaken for future migrations
```

Comment:
I don't think we need this comment on the migration service; just on the db migrate script